### ISSUE TITLE
Admin access

### ISSUE THREAD

**Issue by umaar** [ministryofjustice/fb-editor-install#1]
Is this repo still relevant @solidgoldpig ? If so, could you grant admin access please

**Comment by solidgoldpig** [comment]
I don't think it is relevant any more - the Electron console app has rendered it redundant

I'll let you read it (and its corresponding 20 page help docs) and weep - and then you can delete it ;)
